### DISCUSSION TITLE
爱发电私信不自动回复

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#1529] [提问] [answered]
今天发送更新，获取解锁码都没反应，今天赞助的23.33的

**Comment by cnluminous** [comment]
解锁码不是去qq群获取么

**Comment by Chormore-EDGE** [comment]
![image](https://user-images.githubusercontent.com/114737601/229083627-b1ae6099-d1ce-4ea6-93ea-d89bcc2ea48a.png)

**Comment by WForst-Breeze** [comment, marked as answer]
相关问题请见[ PCL2 常见问题 兼 脑溢血急救中心](https://shimo.im/docs/qKPttVvXKqPD8YDC)

很可能是你已经触发了人工回复，在等待人工回复的过程中无法使用命令。
你如果只是想使用命令，可以发送【取消人工处理】。
